.github/workflows/test.yml

```diff
@@ -1,3 +1,4 @@
+# Run all tests
```

Comment:
Nit: implied by the file name and `name: Tests`, no? 

```suggestion
```

`name: All Tests` could also do the trick. But IMO just naming things "tests" is enough.